plat/imx/common/imx_sip_handler.c

```diff
@@ -168,14 +169,37 @@ int imx_src_handler(uint32_t smc_fid,
 	case IMX_SIP_SRC_IS_SECONDARY_BOOT:
 		val = mmio_read_32(IMX_SRC_BASE + SRC_GPR10_OFFSET);
 		return !!(val & SRC_GPR10_PERSIST_SECONDARY_BOOT);
+#endif /* defined(PLAT_imx8mm) || defined(PLAT_imx8mq) */
+#if defined(PLAT_imx8mm)
```

Comment:
From Imx-atf, imx8mm and imx8mq are same for handle M4

Reply:
`PLAT_imx8mq` is dropped since it is not supported in upstream. See details in commit e3c07d2f5a08 `("docs(imx8m): update build support for imx8mq")`. Therefore, I've also dropped it in this patchset, hence the hadling in different.